documentation/docs/10-prerendering.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Prerendering
+---
+
+It's likely that at least some pages of your app can be represented as a simple HTML file, since they contain no dynamic or user-specific data. These pages can be *prerendered* by your [adapter](#adapters).
+
+If your entire app is suitable for prerendering, you could use [`adapter-static`](https://github.com/sveltejs/kit/tree/master/packages/adapter-static), which will generate HTML files for every page, plus additional files that are requested by `load` functions in those pages.
+
+More likely, you'll only want to prerender specific pages in your app. You'll need to annotate these pages:
+
+```html
+<script context="module">
+	export const prerender = true;
```

Comment:
we should mention somewhere that this needs to be contained in each layout used by a page as well (which would allow us to close https://github.com/sveltejs/kit/issues/386)

Reply:
commented on that issue